cmd/gokr-packer/packer.go

```diff
@@ -281,6 +303,130 @@ func derivePartUUID(hostname string) uint32 {
 	return h.Sum32()
 }
 
+func generateAndSignCert() ([]byte, *rsa.PrivateKey, error) {
+	notBefore := time.Now()
+	// TODO: Make duration configurable?
+	notAfter := notBefore.Add(2 * 365 * 24 * time.Hour)
+	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
+	serialNumber, err := rand.Int(rand.Reader, serialNumberLimit)
+	if err != nil {
+		return nil, nil, err
+	}
+	template := x509.Certificate{
+		SerialNumber: serialNumber,
+		Subject: pkix.Name{
+			//TODO: Configurable?
```

Comment:
Here and above: remove this comment; I don’t think we should make this configurable. If people need more control over their certificates, they should generate thim with another tool.